cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -0,0 +1,1191 @@
+                },
+            },
+            {
+                u"pk": 13,
+                u"model": u"knowledgebase.articlecategorymatrix",
+                u"fields": {
+                    u"created": u"{date}".format(date=self.datetime_now),
+                    u"modified": u"{date}".format(date=self.datetime_now),
+                    u"article": 1,
+                    u"article_category": 13,
+                    u"preferred_signpost": True,
+                },
+            },
+            {
+                u"pk": 14,
+                u"model": u"knowledgebase.articlecategorymatrix",
+                u"fields": {
+                    u"created": u"{date}".format(date=self.datetime_now),
+                    u"modified": u"{date}".format(date=self.datetime_now),
+                    u"article": 1,
+                    u"article_category": 14,
+                    u"preferred_signpost": True,
+                },
+            },
+            {
+                u"pk": 15,
+                u"model": u"knowledgebase.articlecategorymatrix",
+                u"fields": {
+                    u"created": u"{date}".format(date=self.datetime_now),
+                    u"modified": u"{date}".format(date=self.datetime_now),
+                    u"article": 1,
+                    u"article_category": 15,
+                    u"preferred_signpost": False,
+                },
+            },
+            {
+                u"pk": 16,
+                u"model": u"knowledgebase.articlecategorymatrix",
+                u"fields": {
+                    u"created": u"{date}".format(date=self.datetime_now),
+                    u"modified": u"{date}".format(date=self.datetime_now),
+                    u"article": 1,
+                    u"article_category": 16,
+                    u"preferred_signpost": True,
+                },
+            },
+            {
+                u"pk": 17,
+                u"model": u"knowledgebase.articlecategorymatrix",
+                u"fields": {
+                    u"created": u"{date}".format(date=self.datetime_now),
+                    u"modified": u"{date}".format(date=self.datetime_now),
+                    u"article": 1,
+                    u"article_category": 17,
+                    u"preferred_signpost": True,
+                },
+            },
+            {
+                u"pk": 18,
+                u"model": u"knowledgebase.articlecategorymatrix",
+                u"fields": {
+                    u"created": u"{date}".format(date=self.datetime_now),
+                    u"modified": u"{date}".format(date=self.datetime_now),
+                    u"article": 1,
+                    u"article_category": 18,
+                    u"preferred_signpost": False,
+                },
+            },
+        ]
+        outputList = json.loads(outputJSON)
+
+        self.compare_length(outputList, expectedList)
+
+        output_records_in_order = outputList[: len(outputList) - 18]
```

Comment:
You should be able to just use negative indexing here for the slice, ie `[:-18]`

Reply:
Awesome!